en/aggregate-multiple-cluster-monitor-data.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Aggregate Monitoring Data of Multiple TiDB Clusters
+summary: Learn how to aggregate monitoring data of multiple TiDB clusters by Thanos query.
+---
+
+# Aggregate Monitoring Data of Multiple TiDB Clusters
+
+This document describes how to aggregate the monitoring data of multiple TiDB clusters by Thanos to solve the centralization problem of monitoring data of multiple clusters.
+
+## Thanos
+
+[Thanos](https://thanos.io/design.md/) is a high availability solution for Prometheus to simplify the availability guarantee of Prometheus.
+
+Thanos provides [Thanos Query](https://thanos.io/components/query.md/) component as a unified query solution across Prometheus. You can use this feature to solve the problem of aggregating monitoring data of multiple clusters.
+
+## Configure Thanos Query
+
+First, you need to configure a Thanos Sidecar container for each TidbMonitor. To update TidbMonitor, configure Thanos Sidecar and deploy Thanos Query component, refer to [Example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-with-thanos/README.md).In Thanos Query, a Prometheus corresponds to a Store, which corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a unified query interface for monitoring data through Thanos Query's API.
```

Comment:
```suggestion
First, you need to configure a Thanos Sidecar container for each TidbMonitor. Update the `TidbMonitor` CR to configure Thanos Sidecar, and deploy the Thanos Query component. For details, refer to [Example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-with-thanos/README.md).

In Thanos Query, a Prometheus corresponds to a Store, which corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a unified query interface for monitoring data through Thanos Query's API.
```

Reply:
@DanielZhangQD Can we incorporate the [example](https://github.com/pingcap/tidb-operator/tree/master/examples/monitor-with-thanos/README.md) into this doc?

Reply:
First, deploy the TidbMonitor with Thanos Sidecar container:

{{< copyable "shell-regular" >}}

```shell
kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/tidb-monitor.yaml"
```

Then deploy the Thanos Query component:

{{< copyable "shell-regular" >}}

```shell
kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/thanos-query.yaml"
```

In Thanos Query, a Prometheus corresponds to a Store, which corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a unified query interface for monitoring data through Thanos Query's API.

@ran-huang @Joyinqin PTAL.

Reply:
@Joyinqin Can you add this to the Chinese doc? Thank you.